dissect/target/plugins/os/unix/log/lastlog.py

```diff
@@ -1,30 +1,57 @@
-import datetime
-import socket
-import struct
+from typing import BinaryIO
 
 from dissect.target.exceptions import FileNotFoundError
+from dissect import cstruct
 from dissect.target.helpers.record import TargetRecordDescriptor
 from dissect.target.plugin import Plugin, export
-
-from dissect.target.plugins.os.unix.log.utmp import utmp, UtmpFile
-
+from dissect.util import ts
 
 LastlogRecord = TargetRecordDescriptor(
```

Comment:
```suggestion
LastLogRecord = TargetRecordDescriptor(
```
Small naming consistency suggestion. I can't put a suggestion at the `@export` line though, so also rename elsewhere.